internal/core/api.rs

```diff
@@ -287,7 +287,7 @@ pub enum SetRenderingNotifierError {
 /// scene of a component. It provides API to control windowing system specific aspects such
 /// as the position on the screen.
 #[repr(transparent)]
-pub struct Window(pub(crate) WindowInner);
+pub struct Window(pub WindowInner);
```

Comment:
We shouldn't make this public. 

Because that is exposed in your public api, and would let users use out internal API with `window().0.foobar` and we don't want that

Instead of accessing the WindowInner with `.0` you can use `WindowInner::from_pub`